src/app/api/lista-nominal/utils/sorting.ts

```
@@ -0,0 +1,74 @@
+  const [day, month, year] = dateStr.split('/').map((item) => Number(item));
+  // Subtrai 1 porque espera o índice do mês (inicia em 0)
+  const date = new Date(year, month - 1, day);
+  return date;
+};
+
+const compareValues = (
+  a: ValidValue,
+  b: ValidValue,
+  sortOrder: SortOrder
+): number => {
+  const isSortOrderAsc = sortOrder === 'asc';
```

Biome (1.9.4)

[error] Excessive complexity of 16 detected (max: 15).

Please refactor this function to reduce its complexity score from 16 to the max allowed complexity 15.

(lint/complexity/noExcessiveCognitiveComplexity)